docs/source/reference/api.rst

```diff
@@ -1,40 +1,44 @@
 .. _api-reference:
 
-===
 API
 ===
 
 .. automodule:: simphony
-    :members:
-    :inherited-members:
-    :show-inheritance:
+  :members:
+  :inherited-members:
+  :show-inheritance:
 
 .. automodule:: simphony.connect
```

Comment:
`simphony.connect` is an internal module only. We don't really want end-users to know about this module / use it in any way. So let's not include it in the docs.

Reply:
Just removed it.